File: replace.py
import re

file_path = r"c:\Users\tejas\Downloads\Auralix-ads-main\public\portfolio.html"

with open(file_path, "r", encoding="utf-8") as f:
    content = f.read()

def replace_img(match):
    img_tag = match.group(0)
    img_tag = re.sub(r'src="[^"]+"', 'src="HOONY HORIZENTALE.png"', img_tag)
    img_tag = re.sub(r'alt="[^"]+"', 'alt="Upcoming"', img_tag)
    return img_tag

new_content = re.sub(r'<img[^>]+>', replace_img, content)

with open(file_path, "w", encoding="utf-8") as f:
    f.write(new_content)

print("Done")

File: replace.js
const fs = require('fs');
const path = require('path');

const filePath = path.join('c:', 'Users', 'tejas', 'Downloads', 'Auralix-ads-main', 'public', 'portfolio.html');

try {
    let content = fs.readFileSync(filePath, 'utf-8');
    
    // Check if it looks like there is a BOM or it's utf-16
    if (content.charCodeAt(0) === 0xFEFF || content.includes('\0')) {
        content = fs.readFileSync(filePath, 'utf-16le');
    }

    const regex = /<img([^>]+)>/gi;
    const newContent = content.replace(regex, (match, attrs) => {
        let newAttrs = attrs.replace(/src="[^"]+"/, 'src="HOONY HORIZENTALE.png"');
        newAttrs = newAttrs.replace(/alt="[^"]+"/, 'alt="Upcoming"');
        return `<img${newAttrs}>`;
    });

    fs.writeFileSync(filePath, newContent, 'utf-8');
    console.log('Successfully replaced all image sources and alt text.');
} catch (error) {
    console.error('Error processing file:', error);
}
